src/SDKs/DataFactoryV1Extended/Microsoft.Azure.Management.DataFactories.Extended/Microsoft.Azure.Management.DataFactories.Extended.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactories.Extended</PackageId>
+    <Description>Provides DataFactory management capabilities for Microsoft Azure.</Description>
+    <VersionPrefix>4.13.0</VersionPrefix>
```

Comment:
We want to make this assembly as an extension for currently 4.13.0 assembly, so 4.13.0 is actually the start-point for this new assembly.